_events/2021/2021-07-community-call.md

```diff
@@ -0,0 +1,14 @@
+---
+title: July 2021 Community Call
+expires: 2021-07-08
+event_date: "July 8, 2021"
+layout: event
+repeated: false
+---
+
+The next community call will be on July 8, 2021 at 12ET/11CT/10MT/9PT.
+
+This month we are hosting a call on the topic of [Publication Credit for RSEs](https://github.com/USRSE/monthly-community-calls/issues/2). This call will be a chance to explore with others in the US-RSE community how credit can be given to RSEs contributing to research and how RSEs can advocate for themselves to receive credit. Topics such as software publication and citation, financial incentives, acknowledgement, and co-authorship are just a few examples of potential discussion points. As usual, a big part of the discussion will happen in smaller breakout groups. Notes from the call might be incorporated into the white paper resulting from the March community call “Changing how Academia views RSEs.”
+
+Registration details
```

Comment:
This looks intended to be a heading but it just blends into the following sentence when rendered.

Reply:
Ups, yes, I missed that, thanks! I've created a new PR that fixes that: https://github.com/USRSE/usrse.github.io/pull/462